### DISCUSSION TITLE
感觉这个加载很慢

### DISCUSSION THREAD

**Discussion by YyJyyy** [lobehub/lobehub#2639] [Self Hosting] [answered]
感觉这个加载很慢，不知道是缓存问题还是因为服务器的相应慢。在家里使用的时候，点设置面板他都可能要相应30s 才反应出来

[a comment by an automated account is not shown]

**Comment by ozshen** [comment]
似乎是从0.151版本之后代码重构出现的，卡顿现象严重...

[a comment by an automated account is not shown]

**Comment by RubuJam** [comment]
你是使用什么平台部署的，如果是vercel，那么我有些建议或许可以缓解这种情况

[a comment by an automated account is not shown]

**Comment by VectorZhao** [comment]
非常卡顿，换回chatgpt-next-web，你会发现反应速度简直就是光速，对话模型响应也快

[a comment by an automated account is not shown]

**Comment by fanyinghao** [comment]
> 你是使用什么平台部署的，如果是vercel，那么我有些建议或许可以缓解这种情况

@RubuJam 介绍下

[a comment by an automated account is not shown]

**Comment by sagiwei** [comment]
@RubuJam 介绍下经验

[a comment by an automated account is not shown]

**Comment by RubuJam** [comment, marked as answer]
@sagiwei @fanyinghao 可以用下列的API地址代替原本的CNAME地址（如果使用`cloudflare`进行域名管理，请在`cloudflare`的`DNS域名记录`中修改），即`cname.vercel.com`，此时请求会自动访问API对应地址的服务器，而且不影响Vercel管理用此地址绑定的域名。

```
api-yul1.vercel.com
api-arn1.vercel.com
api-hnd1.vercel.com
api-icn1.vercel.com
api-cdg1.vercel.com
api-iad1.vercel.com
api-sin1.vercel.com
api-lhr1.vercel.com
api-oma1.vercel.com
api-bru1.vercel.com
api-cle1.vercel.com
api-dub1.vercel.com
api-gru1.vercel.com
api-bom1.vercel.com
api-lax1.vercel.com
api-chs1.vercel.com
api-hel1.vercel.com
api-zrh1.vercel.com
api-pdx1.vercel.com
api-syd1.vercel.com
api-tpe1.vercel.com
api-sfo1.vercel.com
```
具体命名规则见：
https://vercel.com/docs/edge-network/regions#region-list


The following API addresses can be used to replace the original CNAME address (If you are using `cloudflare` for domain name management, please change this in the `cloudflare DNS domain name record`.) , that is `cname.vercel.com`. These addresses will automatically access the server with the corresponding address of the API, and will not affect Vercel's management of the domain name bound with this address.

```
api-yul1.vercel.com
api-arn1.vercel.com
api-hnd1.vercel.com
api-icn1.vercel.com
api-cdg1.vercel.com
api-iad1.vercel.com
api-sin1.vercel.com
api-lhr1.vercel.com
api-oma1.vercel.com
api-bru1.vercel.com
api-cle1.vercel.com
api-dub1.vercel.com
api-gru1.vercel.com
api-bom1.vercel.com
api-lax1.vercel.com
api-chs1.vercel.com
api-hel1.vercel.com
api-zrh1.vercel.com
api-pdx1.vercel.com
api-syd1.vercel.com
api-tpe1.vercel.com
api-sfo1.vercel.com
```
For further information on the specific naming rules, please refer to the following link:
https://vercel.com/docs/edge-network/regions#region-list

**Comment by GrayXu** [comment]
@RubuJam 即使是访问国内的模型服务提供商，响应速度也很慢。输入到输出第一个字符的延迟非常长。

[a comment by an automated account is not shown]
